steps/cadence-pegasus-fill/configure.yml

```diff
@@ -0,0 +1,55 @@
+#=========================================================================
+# Cadence Pegasus Fill
+#=========================================================================
+# Author : James Thomas
+# Date   : November 2, 2020
+#
+
+name: cadence-pegasus-fill
+
+#-------------------------------------------------------------------------
+# Inputs and Outputs
+#-------------------------------------------------------------------------
+
+inputs:
+  - design.gds
+  - adk
+
+outputs:
+  - fill.gds
+
+#-------------------------------------------------------------------------
+# Commands
+#-------------------------------------------------------------------------
+
+commands:
+  # Generate the dummy fill GDS
+  - pegasus -drc -fill inputs/adk/{fill_rule_deck} -gds design.gds -top_cell {design_name} -dp 8
```

Comment:
What does "-dp 8" do?

Does the fill.gds still have to be merged with the design in a separate node? Maybe put this in a comment at the top above your name/date?

Reply:
-dp 8 runs this with 8 processors. I couldn't find a mode that uses all available processors. Alex was saying we might want to raise this to some high number like 64 so it always uses all of the processors.

Yes, the fill still needs to be merged in a separate node.

Reply:
Maybe make that an "nthreads" parameter like we have elsewhere?

Reply:
Should we use the global nthreads or a fill_nthreads?

Reply:
We should extend whatever we do here to the drc and lvs nodes.

Reply:
I have some concerns about using the nthreads mflowgen parameter here. Particularly, since we're always submitting jobs on VDE, we always wanted to use however many threads are available, regardless of which queue we submit the job to. We could use the `nproc` command to get the number of available cpus.

Reply:
Ok i'm fine doing that - Chris?

Reply:
@jjthomas if not, we could always do that conditionally on the garnet side.